packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -138,11 +141,19 @@ export function buildRootObject(vatPowers, _vatParameters, baggage) {
 
   if (baggage.has('vatAdminDev')) {
     vatAdminDev = baggage.get('vatAdminDev');
-    vatAdminVatID = D(vatAdminDev).getVatAdminVatID();
   }
 
-  const pendingBundles = new Map(); // bundleID -> Promise<BundleCap>
-  const pendingUpgrades = new Map(); // upgradeID -> Promise<UpgradeResults>
+  // This promise is set when service is paused (in which state no service
+  // requests should be accepted unless they can be processed in the same turn)
+  // and cleared when service is resumed.  It is resolved when the vat quiesces,
+  // at which point all pending non-prompt service requests have completed.
+  let quiescentP;
+  let quiescentRR;
+  let amQuiescent = false;
```

Comment:
Yes.  Arguably this could have a better name.  It really means "have reached the quiescent state after being asked to pause" or more precisely "`quiescentP` has been resolved", but neither of those are really things that lend themselves to a tidy name.  I put it in specifically because you can't synchronously poll a promise to see if it's been resolved, but since I was the one resolving it I could take note that I had when I did.  If you have an idea for a better name I'm all ears.